### ISSUE TITLE
Contribute to Emacs?

### ISSUE THREAD

**Issue by phil-s** [iwahbe/jsonian#1]
By default Emacs just uses `javascript-mode` for `.json` files, and the `json-mode` you've referenced is not part of Emacs, so I don't think Emacs has a great out-of-the-box experience for JSON data.

As it sounds like there are benefits to your mode over the existing solutions, please consider contributing it to Emacs.

**Comment by iwahbe** [comment]
I agree that emacs doesn't have good out of the box support for JSON, though it doesn't have good out of the box support for much else either in my opinion. I would be happy to contribute this to emacs, but I think of that more as an end goal. I want to get public feedback and users first. If this package proves useful to people other then me, and once it has been used for a while without major bugs, I will submit a patch to emacs.

**Comment by tecosaur** [comment]
If you want to leave that door open, you'll need to be a tad careful about FSF assignment.
